api/backend/articles/schemas.py

```diff
@@ -0,0 +1,24 @@
+from marshmallow import Schema, fields
+import toastedmarshmallow
+
+# This schema is used to display article data
+class ArticleFormSchema(Schema):
+    title = fields.Str(required=True)
+    cover_image = fields.Str(required=False)
+    content = fields.Str(required=False)
+
+# This schema is used to display article data
+class ArticleSchema(Schema):
+    id = fields.Int(required=True)
+
+    class Meta:
+        # Fields to show when sending data
+        fields = ("id","title", "cover_image", "content","date_published","likes","is_published",)
```

Comment:
Only put id in the fields since that is the only field that we have in ArticleSchema

Reply:
So only display the article id during the GET operation?

Reply:
Yes until i figure out the CDN feature